.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "editor.wordBasedSuggestions": false
+    "editor.wordBasedSuggestions": false,
+    "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
What does this do?

Reply:
Just making sure we are using the same version of typescript?

Reply:
I think this forces VS code to use the version we're using, rather than a different version for type hinting.